### PR TITLE
feat(content-block-card-static): created component

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#5961

### Description
Created `ContentBlockCardStatic` component that contains a static variation of the Card group, a content item, and a button CTA.

### Changelog

**New**

- Content Block Card Static component

### CODE REVIEW

File: packages/web-components/src/components/content-block-card-static/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+
+# `<dds-content-block-card-static>`
```
Comment (jeffchew) [inline packages/web-components/src/components/content-block-card-static/__stories__/README.stories.mdx:5]: @IgnacioBecerra can you also add documentation here about how to enable using the feature flag `DDS_CONTENT_BLOCK_CARD_STATIC`?

**Comment by DragosRistici** [comment]
Hey, Ignatio, thank you so much for this. Looks cool. Quick fixes, please: 

### 1. Cards should be 4 columns instead of 5. 
### 2. Button group - the spacing between buttons should be 16px. 
### 3. Cards should be full width on mobile 
Screenshot 2021-06-04 at 20 29 37

Screenshot 2021-06-04 at 20 29 01

**Comment by DragosRistici** [comment]
A few little things: 

### 1. The CTA type know does not seems to work. 
Screenshot 2021-06-04 at 20 30 52
### 2. Gray 90 and Gray 100 do not have cards border. 
Screenshot 2021-06-04 at 20 32 33

**Review by DragosRistici — APPROVED** [review]
Thank you! Look good!

**Review by jeffchew — APPROVED** [review]
Thank you @IgnacioBecerra !

**Pull request merged**